docs/elasticsearch/cluster.md

```diff
@@ -21,13 +21,17 @@ This is a four-node cluster that has one dedicated master node, one dedicated co
 
 The following table provides brief descriptions of the node types.
 
-Node type | Description
-:--- | :--- |
-`Master` | Manages the overall operation of a cluster and keeps track of the cluster state.
-`Master-eligible` | Available for election to become a new master node.
-`Data` | Stores and searches data.
-`Ingest` | Preprocesses data before storing it in a cluster.
-`Coordinating` | Routes search and aggregation requests.
+Node type | Description | Best practices for production
+:--- | :--- | :-- |
+`Master` | Manages the overall operation of a cluster and keeps track of the cluster state. This includes creating and deleting indices, keeping track of the nodes that join and leave the cluster, checking the health of each node in the cluster (by running ping requests), and allocating shards to nodes. | Three dedicated master nodes in three different zones is right for all/just about all production use cases. This to make sure your cluster never loses quorum. Two of them will be idle for most of the time except when one of them goes down or needs some maintenance.
+`Master-eligible` | Elects one node among them as the master node through a voting process. | Make sure you have dedicated master nodes. The way to achieve a dedicated node type is to mark all other node types as false. In this case, you have to mark all the other nodes as not master-eligible.
```

Comment:
Language here is a bit strong. "For production clusters, make sure you have dedicated master nodes. The way to achieve..."